NDB.Covid19/NDB.Covid19.iOS/Views/BaseViewController.cs

```diff
@@ -56,13 +56,13 @@ void ShowForceUpdatePage(object _)
         /// on the top in View stack that leads to it being read first by VO eventhough a UIAccessibility.PostNotification
         /// is being called to indicate another element that should be in focus. This can be addressed by removing the UIView
         /// that is read first to focus on the element that should be targeted by UIAccessibility.PostNotification call, and
-        /// put it back into accessibility elements after a short delay (0.2 sec or 1 sec for older iOS 12.5 devices).
+        /// put it back into accessibility elements after a short delay (1 sec).
         /// </summary>
         /// <param name="uiViewToDisable">UIView to be removed from VoiceOver reading hierarchy</param>
         protected void removeAccessibilityElementAndEnableAfterDelay(UIView uiViewToDisable)
         {
             uiViewToDisable.IsAccessibilityElement = false;
-            DispatchTime when = new DispatchTime(DispatchTime.Now, TimeSpan.FromSeconds(AppDelegate.ShouldOperateIn12_5Mode? 1.0 : 0.2));
+            DispatchTime when = new DispatchTime(DispatchTime.Now, TimeSpan.FromSeconds(1));
```

Comment:
just asking... can it cause issues with jumping focus if somebody will focus different object before this delay?

Reply:
it is a different object that gets focused, that's the point of setting a delay so the element pointed out by UIAccessibility notification will get focused, and voiceover won't just jump to the X button. Please, feel free to elaborate, unsure I am getting the question.

Reply:
as i remember when i had been developing the timed reminder feature with custom dialog i had problems with focus in accessibility. sometimes it was focusing the last element in the view and then after specified delay (just like here) it was jumping to the element specified by me. But Voice Over had been reading both elements.

Reply:
as i can see i had to make all other elements hidden with:
View.AccessibilityElementsHidden = true
until specified time pass and then make them visible again.

Reply:
It is enough only to set `isAccessibilityElement = false` because it will make the element fully not accessible by VoiceOver.

Reply:
For me it was not enough. but this case may be different so i believe :)

Reply:
If you are taking about a container view that in itself contains more accessibility elements, then perhaps `AccessibilityElementsHidden = true` makes sense.